docs/guides/scripts.md

```diff
@@ -62,24 +64,30 @@ main()
     console.error(error);
     process.exit(1);
   });
+
 ```
 
 And there you can see how the [Buidler Runtime Environment] is accessed at the top, which makes this script work in a standalone fashion:
 
 ```
 $ node scripts/sample-script.js
 All contracts have already been compiled, skipping compilation.
-Greeter address: 0x494d39079b81c620c0ebea503b9295331bfc34c2
+Greeter address: 0x7c2C195CD6D34B8F845992d380aADB2730bB9C6F
 ```
 
 But the script can also run through Buidler:
 
 ```
-$ npx buidler run scripts/sample-script.js
+$ npx buidler run scripts/sample-script.js 
+All contracts have already been compiled, skipping compilation.
 All contracts have already been compiled, skipping compilation.
-Greeter address: 0x494d39079b81c620c0ebea503b9295331bfc34c2
+Greeter address: 0x7c2C195CD6D34B8F845992d380aADB2730bB9C6F
 ```
 
+::: tip
+Did you notice the double compile message? When running a script through `npx buidler run`, the `compile` task will be called before running the script, but you can skip this with the `--no-compile` parameter.
```

Comment:
I wouldn't remove `bre.run('compile')` because it's used as example on how to use Buidler as a library making the script standalone.  Maybe we could pick another `task` instead of `compile` to use in the sample script. 

I honestly prefer `compile` + the `::: tip :::`.

Reply:
I see that, but invoking the `compile` task to yield two compilations and then tell the user to use the `--no-compile` arg to undo our `compile` call by disabling the default behavior of compiling is just really convoluted.

@alcuadrado any thoughts?

Reply:
Why not just leave the `bre.run` as a comment?

Reply:
It works as an explanation and doesn't do the double compilation thing.

Reply:
because the line is needed for the standalone script. 
- leave the call to `bre.run` in and simply add a step to comment it out before running the script through Buidler, explaining that that's not needed when running scripts through buidler
- remove the tip block

Reply:
I removed the tip block but left the call to `bre.run` commented because the script is also mentioned on the Getting Started section. Also added a step to uncomment this line out to explain how to manually compile.